src/nvim/lib/kvec.h

```diff
@@ -134,6 +134,9 @@ static inline void *_memcpy_free(void *const restrict dest,
 
 /// Resize vector with preallocated array
 ///
+/// @note May not resize to an array smaller then init_array: if requested,
+///       init_array will be used.
```

Comment:
@justinmk “Ignored” in your variant tells nothing about what `kvi_resize` will actually do.